x-pack/plugins/siem/public/components/page/hosts/host_overview/index.tsx

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiDescriptionList, EuiFlexItem } from '@elastic/eui';
+import { getOr } from 'lodash/fp';
+import React from 'react';
+import { pure } from 'recompose';
+import { HostItem } from '../../../../graphql/types';
+import { getEmptyTagValue } from '../../../empty_value';
+
+import * as i18n from './translations';
+import { FirstLastSeenHost, FirstLastSeenHostType } from '../first_last_seen_host';
+import { DefaultFieldRenderer, hostIdRenderer } from '../../../field_renderers/field_renderers';
+import { LoadingPanel } from '../../../loading';
+import { LoadingOverlay, OverviewWrapper } from '../../index';
+import { IPDetailsLink } from '../../../links';
+
+interface DescriptionList {
+  title: string;
+  description: JSX.Element;
+}
+
+interface OwnProps {
+  data: HostItem;
+  loading: boolean;
+}
+
+type HostSummaryProps = OwnProps;
+
+const getDescriptionList = (descriptionList: DescriptionList[], key: number) => {
+  return (
+    <EuiFlexItem key={key}>
+      <EuiDescriptionList listItems={descriptionList} />
+    </EuiFlexItem>
+  );
+};
+
+export const HostOverview = pure<HostSummaryProps>(({ data, loading }) => {
+  const getDefaultRenderer = (fieldName: string, fieldData: HostItem) => (
+    <DefaultFieldRenderer
+      rowItems={getOr([], fieldName, fieldData)}
+      attrName={fieldName}
+      idPrefix="host-overview"
+    />
+  );
+
+  const descriptionLists: Readonly<DescriptionList[][]> = [
+    [
+      {
+        title: i18n.HOST_ID,
+        description: data && data.host ? hostIdRenderer(data.host) : getEmptyTagValue(),
```

Comment:
Don't know if you need the `data &&` as you don't check for its existence below. This can be `data.host ?`

Reply:
Nice catch! `HostItem` can't be null as typed below, so this check isn't necessary.

``` ts
export interface HostsEdges {
  node: HostItem;
  cursor: CursorType;
}
```